web/react/components/search_results.jsx

```diff
@@ -64,6 +65,11 @@ SearchItem = React.createClass({
                 dispatchError(err, "getPost");
             }
         );
+
+    var postChannel = ChannelStore.get(this.props.post.channel_id);
+    var teammate = postChannel.type === 'D' ? UserStore.getProfile(postChannel.name.substring(postChannel.name.indexOf('_')+2)).username : "";
```

Comment:
There isn't any guarantee that first id is the other user's id since they are just alpha sorted, it may be your id.  Check with Joram about that method (it might be backend only) if so we may need to port to the client.